Gemfile

```diff
@@ -166,6 +166,7 @@ group :development, :test do
   gem 'awesome_print'
   gem 'puma', '~> 5.3'
   gem 'debug'
+  gem 'bootsnap', '~> 1.13.0', require: 'bootsnap/setup'
```

Comment:
[The documentation for bootsnap](https://github.com/Shopify/bootsnap#usage) suggests loading bootsnap in the Gemfile as an alternative to loading it in `config/boot.rb`. `Rails.env.development?` and `Rails.env.test?` cannot be accessed in `config/boot.rb`, which is why I've put it in the Gemfile.

Reply:
Sounds good - we can keep it simple for now. 

As this is only being used in test/dev environments, we can relax the pessimistic versioning. Let's set it to `'~> 1.13'`.